app/assets/sass/modules/_web-payments.scss

```diff
@@ -1,20 +1,35 @@
-#apple-pay-container,
-#google-pay-container {
+.apple-pay-container{
   display: none;
-}
 
-// Toggle Apple Pay
-.apple-pay-available {
-  #apple-pay-container {
+  .apple-pay-available & {
     display: block;
   }
 }
 
-// Toggle Google Pay
-.google-pay-available {
-  #google-pay-container {
+.google-pay-container {
+  display: none;
+
+  .google-pay-available & {
     display: block;
-  }
+  } 
+}
+
+.web-payment-button-heading{
+  display: none;
+  
+  .apple-pay-available &,
+  .google-pay-available & {
+    display: block; 
```

Comment:
Using a SCSS Parent selector - only show these headings when the body has `apple-pay-available` or `google-pay-available` added to it.
`apple-pay-available` and/or `google-pay-available` CSS classes are added by JS in the browser if the browser supports Apple Pay.